force-app/components/courseInvitation/lwc/courseInvitation/courseInvitation.html

```diff
@@ -48,10 +48,10 @@
 
       <!-- CONFIRM -->
       <lightning-button
+        if:false={falseMoreThanTenAndEmailSent}
```

Comment:
```suggestion
        if:false={emailSent}
```

endre tilbake til emailSent, fordi import skjules midlertidig